src/olympia/abuse/management/commands/retry_unreported_abuse_reports.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand
+
+import olympia.core.logger
+from olympia.abuse.models import AbuseReport, AbuseReportManager
+from olympia.abuse.tasks import report_to_cinder
+
+
+class Command(BaseCommand):
+    log = olympia.core.logger.getLogger('z.abuse')
+
+    def handle(self, *args, **options):
+        qs = AbuseReport.objects.filter(
+            AbuseReportManager.is_individually_actionable_q(), cinder_job__isnull=True
+        )
```

Comment:
Add `.values_list('pk', flat=True)` IMHO. That makes the query slightly less nasty and since you're fetching it all in one go, consume less memory.

We don't care about the str representation of each report in the `log.info()` call below, logging pks will be enough. You'll need to adjust the `.delay()` call since the queryset would then only contain pks though.

Reply:
https://github.com/mozilla/addons-server/pull/23108/commits/6544c582bf34b14838991da1fefd95f6318a3fd4